Userland/Libraries/LibWeb/InProcessWebView.cpp

```diff
@@ -273,7 +273,7 @@ void InProcessWebView::keydown_event(GUI::KeyEvent& event)
         }
     }
 
-    event.accept();
+    event.ignore();
```

Comment:
This seems at odds with `page_accepted_event` above. If the page accepted the event, we shouldn't call `ignore()` on it.